ztest/ztest.go

```diff
@@ -135,11 +136,28 @@ func Run(t *testing.T, dirname string) {
 // ZTest defines a ztest.
 type ZTest struct {
 	ZQL          string `yaml:"zql"`
-	Input        string `yaml:"input"`
+	Input        Inputs `yaml:"input"`
 	OutputFormat string `yaml:"output-format,omitempty"`
 	Output       string `yaml:"output"`
 }
 
+type Inputs []string
+
+func (i *Inputs) UnmarshalYAML(value *yaml.Node) error {
```

Comment:
Nit: Here and elsewhere, please add doc strings to the methods and functions you introduce. Since the rest of the file already has them, let's keep the convention going. Refer to https://golang.org/doc/effective_go.html#commentary as a reminder.

Reply:
I'm going to try to make this private- it's not really something an end user would/should care about.